HelpAssistHer/help-assist-her
# Update hours for verification portal

## Description
I updated the front-end code to display the open and close hours for a PRC. In doing so, I also made a couple of minor changes that I think will really help with readability of this code.

1. I know we originally discussed storing the days of the week as numbers. I think the reason was because that's how Google stores the hours? While I was writing this code though, I realized that's actually quite confusing. It's not clear whether '0' is Sunday or Monday. For the sake of our own memories and for future developers that may work on this code, I think it's much easier, more clear, and lends toward cleaner code to just use the full name of the day, always lower case. And even if Google uses numbers, I think it will be super easy to write a simple transformation function to convert Google's keys to our format. 

2. I believe we originally discussed storing the hours as the number of seconds from midnight. To make this simpler, we can actually just use the minutes from midnight. A PRC (or any business for that matter) will never open at a time with seconds: i.e. `08:00:52`. 😄 While this isn't a huge change, the numbers stored will be smaller and are slightly easier to manage. We also won't have to worry about scenarios where the number is stored as something that would indicate 30 seconds into a minute (`08:00:30`) and then have to deal with weird UI issues displaying that time (is it `08:00` or `08:01`?

Check out the changes in this PR and if you have any concerns with this approach, let's discuss in more detail so we're both on the same page before merging this PR.

```
"hours" : {
        "sunday" : {
            "closedAllDay" : true
        },
        "monday" : {
            "open" : 360,
            "close" : 1080,
            "closedAllDay" : false
        },
        "tuesday" : {
            "open" : 360,
            "close" : 1110,
            "closedAllDay" : false
        },
        "wednesday" : {
            "open" : 360,
            "close" : 1080,
            "closedAllDay" : false
        },
        "thursday" : {
            "open" : 360,
            "close" : 1080,
            "closedAllDay" : false
        },
        "friday" : {
            "open" : 360,
            "close" : 1080,
            "closedAllDay" : false
        },
        "saturday" : {
            "closedAllDay" : true
        }
    }
```

## Test Plan
1. In Robomongo, update the PRC that will load when you click 'Get One Resource' with the hours data from above
1. Check out this branch 
1. Click Get One Resource
1. Verify the correct hours loaded into the correct fields for each day of the week
